BabelfishFeatures.cfg

```diff
@@ -1096,6 +1100,8 @@ supported-3.5.0-3.*=sp_procedure_params_100_managed
 supported-4.1.0=sp_procedure_params_100_managed
 supported-4.2.0=sp_renamedb
 supported-4.4.0=sp_reset_connection
+supported-4.7.0=sp_helplogins

```

Comment:
`supported-4.7.0=sp_helplogins` means that this is supported in all subsequent releases, so that includes 5.1 and 5.2. Is that correct? (I think not) If not, it should be `supported-4.7.0-4.*=sp_helplogins`